src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsPage.js

```diff
@@ -63,12 +70,14 @@ const IOUParticipantsPage = (props) => {
                 onStepComplete={props.onStepComplete}
                 participants={props.participants}
                 onAddParticipants={props.onAddParticipants}
+                safeAreaPaddingBottomStyle={props.safeAreaPaddingBottomStyle}
```

Comment:
Good catch. We don't need to pass `safeAreaPaddingBottomStyle` prop to `IOUParticipantsSplit`.
But should pass to IOUParticipantsRequest unless it's wrapped with `SafeAreaConsumer`.